File: Examples/CSharp/Loading-and-Saving/SaveWithDefaultSettings.cs
using System.IO;
using Aspose.Note;
using System;

namespace Aspose.Note.Examples.CSharp.Loading_Saving
{
    public class SaveWithDefaultSettings
    {
        public static void Run()
        {
            // ExStart:SaveWithDefaultSettings
            // ExFor:Document
            // ExFor:Document.Save(System.IO.Stream, Aspose.Note.SaveFormat)
            // ExFor:SaveFormat
            // ExSummary:Shows how to save a document in pdf format using default settings.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            // Load the document into Aspose.Note.
            Document oneFile = new Document(dataDir + "Aspose.one");

            // Save the document as PDF
            dataDir = dataDir + "SaveWithDefaultSettings_out.pdf";
            oneFile.Save(dataDir, SaveFormat.Pdf);
            
            // ExEnd:SaveWithDefaultSettings
            
            Console.WriteLine("\nOneNote document saved successfully with default settings.\nFile saved at " + dataDir); 
        }
    }
}

File: Examples/VisualBasic/Loading-and-Saving/SaveWithDefaultSettings.vb
Imports System.IO
Imports Aspose.Note

Namespace Loading_Saving
    Public Class SaveWithDefaultSettings
        Public Shared Sub Run()
            ' ExStart:SaveWithDefaultSettings
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadingAndSaving()

            ' Load the document into Aspose.Note.
            Dim oneFile As New Document(dataDir & Convert.ToString("Aspose.one"))

            dataDir = dataDir & Convert.ToString("SaveWithDefaultSettings_out.pdf")
            ' Save the document as PDF
            oneFile.Save(dataDir, SaveFormat.Pdf)
            ' ExEnd:SaveWithDefaultSettings
            Console.WriteLine(Convert.ToString(vbLf & "OneNote document saved successfully with default settings." & vbLf & "File saved at ") & dataDir)
        End Sub

    End Class
End Namespace
